tests/translator/input/error_application_properties.yaml

```diff
@@ -33,4 +33,13 @@ Resources:
     Properties:
       Location:
         ApplicationId:
-        SemanticVersion:
\ No newline at end of file
+        SemanticVersion:
+
+  IntrinsicProperties:
+    Type: 'AWS::Serverless::Application'
+    Properties:
+      Location:
+        ApplicationId:
+          Sub: foobar
```

Comment:
Could make one of these be `Fn::Sub` to test for unsupported intrinsics as well as random dict keys.